partials/footer.html

```diff
@@ -37,7 +37,7 @@
     <!-- JavaScript
     ================================================== -->
 
-    <script src="main.js"></script>
+    <script src="/main.js"></script>
```

Comment:
to allow pages like "gamevy.com/folder/something.html" to fetch the js from "gamey.com/main.js" instead of "gamevy.com/folder/main.js". I should have done it like this from the beginning.